src/types/Tracking.ts

```diff
@@ -36,3 +36,22 @@ export type MonitoringPlotSearchResult = {
 // planting seasons
 export type PlantingSeason = components['schemas']['PlantingSeasonPayload'];
 export type UpdatedPlantingSeason = components['schemas']['UpdatedPlantingSeasonPayload'];
+
+export type Location = {
+  timeZone?: string;
+};
+
+export type SitePlantingZone = Omit<PlantingZone, 'areaHa' | 'plantingSubzones'> & {
+  plantingSubzones: Omit<PlantingSubzone, 'areaHa'>[];
+};
+
+export type SiteDetails = Location & {
```

Comment:
This is safe with current state/plans. If PlantingSite and DraftPlantingSite diverge dramatically at some point, we can split things out separately.